besu/src/test/java/org/hyperledger/besu/cli/options/EthProtocolOptionsTest.java

```diff
@@ -18,7 +18,7 @@
 import static org.mockito.Mockito.verifyNoInteractions;
 
 import org.hyperledger.besu.ethereum.eth.EthProtocolConfiguration;
-import org.hyperledger.besu.util.number.PositiveNumber;
+import org.hyperledger.besu.ethereum.eth.ImmutableEthProtocolConfiguration;
```

Comment:
do you need to run the annotation processor over EthProtocolConfiguration prior to writing this file? (i.e.to ensure ImmutableEthProtcolConfiguration exists)?

Does Auto-import etc. still work in the IDE?

Reply:
You simply run your build or your tests and missing classes will be generated if needed. You might have an error the first time you run the build. But after that everything works fine, including auto import and all other features.